docs/enhancement-proposals/off-ep-0005.md

```diff
@@ -0,0 +1,123 @@
+should be used for periodic (i.e. condensed-phase) and non-periodic (i.e. gas phase) systems. This
+in particular enables a force field to specify that PME should be used for periodic systems and no
+cutoff should be used for non-periodic systems.
+
+## Motivation and Scope
+
+In version 0.3 of the `<Electrostatics>` tag, the default and most commonly-used electrostatics
+method is PME (`method="PME")`. It is, however, not compatible with non-periodic systems, such as
+gas-phase and/or single-molecule systems. In practice, force fields defined with PME are modified by
+users in ways that make them compatible with non-periodic systems, such as the
+[``NoCutoff``](http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html?highlight=nocutoff#coulomb-interaction-without-cutoff)
+option provided by OpenMM's ``NonbondedForce``.
+
+Given that is it not tractable to use (much less train) a force field that can only be applied to
+periodic OR non-periodic systems, this OFF-EP proposes that the `<Electrostatics>` tag may specify
+that different methods should be used for each kind of systems.
+
+This OFF-EP
+* does not propose changes any detail of the `<vdW>` section
+* does not alter the intended meaning of `method="PME"`
+* does not add allowed methods for non-periodic systems (still only `Coulomb`), but does require
+  explicitly specifying what is used for non-periodic systems
+
+Each of the above points are important and worthy of consideration but should be scoped to separate OFF-EPs.
+
+## Usage and Impact
+
+Since most force fields use some flavor of PME for periodic systems and something similar to
+`method_nonperiodic="Coulomb"` for non-periodic systems, the default attributes for this tag will
+likely be the most commonly-used. Splitting `method` out into two attributes, however, makes it less
+ambiguous how electrostatics should be handled in each case and decouples the method used in each
+case. Users are recommended to consider upgrading from the default attribute values of 0.3 to 0.4 to avoid
+continuing to use this ambiguity. Implementations may wish to execute this up-conversion
+automatically (see below).
+
+The impact to a vast majority of users should be limited to needing to modify this line in their
+force field files. Non-standard electrostatics settings are not generally covered here and should be
+the focus of future OFF-EPs.
+
+## Backward compatibility
+
+Version 0.4 of this tag is backwards-incompatible with older versions because the information
+content is different. It is ambiguous how to map both `method_periodic` and `method_nonperiodic`
+back down to a single `method` and it is not recommended to try doing so.
+
+Implementations may wish to add up-converters from old versions. A common up-converter could convert the
+following tag header
+
+```
+<Electrostatics version="0.3" method="PME" scale12="0.0" scale13="0.0" scale14="0.833333" scale15="1.0"/>
+```
+
+to the equivalent of reading
```

Comment:
Not sure this is grammatically correct; should it be "to the less ambiguous new version, which would read" or something? It depends on what you mean here and i find it at least confusing and possibly not grammatically correct.

Reply:
I'm more or less trying to say how the toolkit will/should handle up-conversions of this snippet, which I'm pretty sure is the only one that's out in the wild. I can see I was being a little ambiguous in thinking about the in-memory representation where simply saying what is should look like post up-conversion would be simpler.